yb-voyager/cmd/templates/migration_assessment_report.template

```diff
@@ -516,59 +516,109 @@
             </p>
         {{ end }}
 
-        {{ $allIssuesWithoutPerfOptimzations := filterOutPerformanceOptimizationIssues .Issues }}
-        <h2>Assessment Issues</h2>
-        {{ if $allIssuesWithoutPerfOptimzations }}
+        {{ template "issuesTable" (dict "onlyPerf" false "Issues" .Issues) }}
+        
+        {{ template "issuesTable" (dict "onlyPerf" true "Issues" .Issues) }}
+        
+{{ define "issuesTable" }}
+  {{ $heading := "Assessment Issues" }}
+  {{ $issues := filterOutPerformanceOptimizationIssues .Issues }}
+
+  {{ $totalCountKeyWord := "Total issues" }}
+
+  {{ $issueKeyword := "Issue" }}
+
+  {{ $expandButton := "expandAllButton" }}
+  {{ $collapseButton := "collapseAllButton" }}
+
+
+  {{ if .onlyPerf }}
+    {{ $heading = "Performance Optimzations" }}
+    {{ $issues = getPerformanceOptimizationIssues .Issues }}
+    {{ $totalCountKeyWord = "Total optimizations" }}
+    {{ $issueKeyword = "Optimization" }}
+    {{ $expandButton = "expandAllButton-perf" }}
+    {{ $collapseButton = "collapseAllButton-perf" }}
+
+  {{ end }}
+
+        {{ if $issues }}
```

Comment:
Order of if-else conditions: smaller case can be written and get out of way first, here when issues.len == 0

Reply:
I think this way it's clearer here what all checks are there, else we need to add more if-elif-conditions to have the same UI